daemon/loadbalancer.go

```diff
@@ -246,6 +246,17 @@ func (d *Daemon) svcDelete(svc *loadbalancer.LBSVC) error {
 }
 
 func (d *Daemon) svcDeleteBPF(svc loadbalancer.L3n4AddrID) error {
+	if err := lbmap.DeleteServiceV2(svc, service.DeleteBackendID); err != nil {
```

Comment:
If this fails, then the deletion of the legacy service will not occur - why not delete both, and synthesize an error in the case that only one failed, or if both failed to be deleted?

Reply:
Sure, good suggestion. Changed.